### PR TITLE
Add Kafka Topic creation in local mode

### PR DESCRIPTION
**Pull request by psav**
* Kafka will now create topics in local mode
* An additional port-forward is required to connec to kafka

### CODE REVIEW

**Comment by kylape** [comment]
```
build command-line-arguments: cannot find module for path github.com/confluentinc/confluent-kafka-go/kafka/librdkafka
```

**Comment by kylape** [comment]
That was much easier than I expected

File: controllers/cloud.redhat.com/providers/kafka/localkafka.go [hunk 1]
```diff
@@ -43,6 +46,12 @@ func (k *localKafka) CreateTopics(app *crd.ClowdApp) error {
 				RequestedName: topic.TopicName,
 			},
 		)
+
+		conn, err := kafka.DialLeader(k.Ctx, "tcp", host, topicName, 0)
```
Comment (bsquizz) [inline controllers/cloud.redhat.com/providers/kafka/localkafka.go:50]: I am guessing this results in a single partition on the topic, right? Do we need to go with the second approach here so we can create the topic with the explicit number of partitions specified in our `ClowdEnvironment KafkaTopicSpec`?

https://github.com/segmentio/kafka-go#to-create-topics

**Pull request merged**